live-examples/html-examples/table-content/css/caption.css

```diff
@@ -0,0 +1,33 @@
+caption {
+    padding: 10px;
+    caption-side: bottom;
+}
+
+table {
+    border-collapse: collapse;
+    border: 2px solid rgb(200, 200, 200);
+    letter-spacing: 1px;
+    font-family: sans-serif;
+    font-size: 0.8rem;
+}
+
+td, th {
+    border: 1px solid rgb(190, 190, 190);
+    padding: 10px 20px;
```

Comment:
I think we should reduce horizontal padding to 10px, otherwise the table overflows horizontally at narrower widths.